query_execution/QueryContext.proto

```diff
@@ -52,4 +52,6 @@ message QueryContext {
 
   // NOTE(zuyu): For UpdateWorkOrder only.
   repeated UpdateGroup update_groups = 9;
+
+  required uint64 query_id = 10;
```

Comment:
`query_id` will not change in future, but some fields may add or remove. So setting `query_id` to `1` reduce the chances to change the assigning numbers.